azure/api/v1alpha1/account.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	log = logging.Logger.WithName(Group)
+	log = logging.Logger.WithName(StorageGroupVersion.Group)
```

Comment:
Unrelated to this PR, but I think this may be the only place that we have a logger in an API package. Might be a good idea to move it to the controller by returning an error in `parseStorageAccountSpec()` if JSON unmarshaling fails, then pass that error along to the claim controller via `ParseAccountSpec()`. However, the need for all of these helper functions should be eliminated by strongly typed resource classes.